gno.land/pkg/gnoclient/errors.go

```diff
@@ -0,0 +1,10 @@
+package gnoclient
+
+import "errors"
+
+var (
+	errInvalidPkgPath   = errors.New("invalid pkgpath")
+	errInvalidFuncName  = errors.New("invalid function name")
+	errMissingSigner    = errors.New("missing Signer")
+	errMissingRPCClient = errors.New("missing RPCClient")
+)
```

Comment:
I don't like how specific functionality errors (`Call` errors) are bundled with general errors for the actual client.

I propose you return these errors at their origin (not in a separate file like this), and separate them out into different `var` groups

Reply:
I returned them to the top of `client_txs.go.